maven-archetype-quickstart/src/main/resources-filtered/archetype-resources/pom.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-#macro( compilerProperties $jcv )
+#macro( compilerProperties $javaCompilerVersion )
```

Comment:
please do not revert it. As explained and agreed in: https://github.com/apache/maven-archetypes/pull/18#discussion_r1432919700

Reply:
commit reverted - please open new with all correct changes